frontend/packages/client/src/components/ProposalInformation/Information.js

```diff
@@ -0,0 +1,150 @@
+              id={communityId}
+              logo={communityLogo}
+            />
+            <Link to={`/community/${communityId}`}>
+              <p className="small-text px-2 is-underlined has-text-grey">
+                {communityName}
+              </p>
+            </Link>
+          </div>
+        }
+      />
+
+      <InfoBlock
+        title={'Proposed by'}
+        component={
+          <BlockieWithAddress
+            creatorAddr={creatorAddr}
+            isCoreCreator={isCoreCreator}
+          />
+        }
+      />
+      <InfoBlock
+        title="Voting strategy"
+        component={
+          <div className="is-flex" onClick={openStrategyModal}>
+            <div className="pr-2 cursor-pointer has-text-grey">
+              {strategyName}
+            </div>
+          </div>
+        }
+      />
+      <InfoBlock
+        title={'Token required'}
+        component={
+          <a
+            href={'https://flowscan.org/'}
+            rel="noopener noreferrer"
+            target="_blank"
+            className="button is-text p-0 small-text"
+            style={{ height: '2rem !important' }}
+          >
+            <p className="mr-2">{`$${tokenName?.toUpperCase()}`}</p>
+            <Svg name="LinkOut" width="12" height="12" />
+          </a>
+        }
+      />
+      <InfoBlock
+        title={'Max tokens'}
+        component={<span className="has-text-grey">{maxWeight}</span>}
+      />
+      <InfoBlock
+        title={'Min tokens'}
+        component={<span className="has-text-grey">{minBalance}</span>}
+      />
+
+      <InfoBlock
+        title={'Start date'}
+        component={
+          <span className="has-text-grey">
+            {parseDateFromServer(startTime).date.toLocaleString(
+              undefined,
+              dateFormatConf
+            )}
+          </span>
+        }
+      />
+      <InfoBlock
+        title={'End date'}
+        component={
+          <span className="has-text-grey">
+            {parseDateFromServer(endTime).date.toLocaleString(
+              undefined,
+              dateFormatConf
+            )}
+          </span>
+        }
+        isLastElement={!ipfs}
+      />
+      {ipfs && (
+        <InfoBlock
+          title={'IPFS'}
+          component={
+            <a
+              href={ipfsUrl}
+              rel="noopener noreferrer"
+              target="_blank"
+              className="button is-text p-0 small-text"
+              style={{ height: '2rem !important' }}
+            >
+              <Tooltip
+                classNames="is-flex is-flex-grow-1 is-align-items-center"
+                position="top"
+                text="Open Ipfs link"
+              >
+                <p className="mr-2">{`${ipfs.substring(0, 8)}`}</p>
+                <Svg name="LinkOut" width="12" height="12" />
+              </Tooltip>
+            </a>
+          }
+          isLastElement
+        />
+      )}
+    </div>
+  );
+}
```

Comment:
this was part of the ProposalInformation component, but since that component has a lot of logic for sticking the panel and also loading data, this one was created to show data only